examples/node/src/main.rs

```diff
@@ -0,0 +1,145 @@
+use std::net::{AddrParseError, SocketAddr};
+
+use drop::crypto::key::exchange::{Exchanger, KeyPair, PublicKey};
+use drop::crypto::sign::{self, Signer};
+use drop::crypto::ParseHexError;
+use drop::net::{TcpConnector, TcpListener};
+use drop::system::manager::{Handle, SystemManager};
+use drop::system::sampler::AllSampler;
+use drop::system::System;
+
+use murmur::{Fixed, Payload, Sequence, Sieve, SieveConfig};
```

Comment:
```suggestion
use sieve::{Fixed, Payload, Sequence, Sieve, SieveConfig};
```

and `sed s,murmur,sieve,g` for the whole file